netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -127,6 +127,23 @@ public static NettyChannelBuilder forAddress(SocketAddress serverAddress) {
     return new NettyChannelBuilder(serverAddress);
   }
 
+  /**
+   * Creates a new builder with the given server address. This factory method is primarily intended
+   * for using Netty Channel types other than SocketChannel. {@link #forAddress(String, int)} should
```

Comment:
Looks like this link should be updated: `forAddress(String, int, ChannelCredentials)`

Reply:
Oops! Now fixed.